README.md

```diff
@@ -1,2 +1,21 @@
 # smile-warrior
 This repo contains code used for smile detector for project of Gradient Science Club.
+
+## To create dataset:
+Run:
+```
+   python adjust_dataset.py 
+```
+
+Result will be csv file named 'smile_warrior_dataset'.
+
+## To load dataset:
+Use function:
+```
+    load_dataset  
```

Comment:
As I said already - do not mention functions you created in README. It should be transparent to the user which functions we use in training. If user wants to use the function, he will go to see the commentary within that file.

Reply:
Ok